index.js

```diff
@@ -262,6 +274,8 @@ function Argv (processArgs, cwd) {
         self.default(key, opt.default)
       } if ('nargs' in opt) {
         self.nargs(key, opt.nargs)
+      } if ('choices' in opt) {
```

Comment:
What features are missing here? Any related issues for them? Anything different I should do here?